cmd/parca-agent/main.go

```
@@ -390,16 +268,10 @@ func main() {
 		})
 	}
 
-	// Run group for target manager
-	{
-		ctx, cancel := context.WithCancel(ctx)
-		g.Add(func() error {
-			level.Debug(logger).Log("msg", "starting target manager")
-			return tm.Run(ctx, m.SyncCh())
-		}, func(error) {
-			cancel()
-		})
-	}
+	// Add profilers.
+	_ = pp.AddProfiler(ctx, profiler.NewCPUProfiler, func() map[int]model.LabelSet {
```

Let's not just ignore the errors.